src/main/java/sg/ncl/exceptions/DeterLabOperationFailedException.java

```diff
@@ -0,0 +1,12 @@
+package sg.ncl.exceptions;
+
+import sg.ncl.domain.ExceptionState;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'sg.ncl.domain.ExceptionState'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3AUselessImportCheck)